lib/stats/DefaultStatsPrinterPlugin.js

```diff
@@ -1128,7 +1185,11 @@ class DefaultStatsPrinterPlugin {
 									}
 								}
 								if (start) {
-									context[color] = str => `${start}${str}\u001b[39m\u001b[22m`;
+									context[color] = str =>
+										`${start}${str.replace(
+											/((\u001b\[39m|\u001b\[22m|\u001b\[0m)+)/g,
+											`$1${start}`
+										)}\u001b[39m\u001b[22m`;
```

Comment:
I think regression of this change - https://github.com/webpack/webpack/issues/12608